TurkishFilter/sections/specific.txt

```diff
@@ -64,6 +64,8 @@ t54.com.tr,haber266.com,safihaber.com,nazilliadalet.com,bizimyaka.com,spor46.com
 ||yeniasya.com.tr/sites/yeniasya/static/*-kampanya
 !
 ||guvenlifilmizle.com/header.gif
+halklailiskiler.com.tr###leader-wrapper
+/\/images\/uploads\/ad[0-9][0-9][0-9][0-9].(gif)/$domain=halklailiskiler.com.tr,image
```

Comment:
It's better to impove it to this `/halklailiskiler\.com\.tr\/images\/uploads\/ad[0-9]+\.gif/`